### PR TITLE
Upgrade gems and fix styling issues

### PR DESCRIPTION
**Pull request by wallin**
- Remove gem locks, and bring them up to date
- Drop support for Ruby 2.0
- Fix trivial styling issues as per Rubocop

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Documentation:
```
Comment (mastahyeti) [inline .rubocop.yml]: Does anything need to be changed for rubocop to run as a part of CI?

Comment (wallin) [reply .rubocop.yml]: Not really, I'm running it through coditsu.io now which post back status. This file is mostly for local development

**Comment by nijikon** [comment]
You can drop support for 2.1 as well; it's not supported by upstream anyway.

**Comment by wallin** [comment]
@nijikon ok. Will do that in a separate PR

**Pull request merged**